lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -383,18 +384,19 @@ def add_creditcard(post, creditcard, options)
         card = {}
         if emv_payment?(creditcard)
           add_emv_creditcard(post, creditcard.icc_data)
-          post[:card][:read_method] = "contactless" if creditcard.contactless_emv
-          post[:card][:read_method] = "contactless_magstripe_mode" if creditcard.contactless_magstripe
+          post[:card][:read_method] = "contactless" if creditcard.read_method == 'contactless'
```

Comment:
For normal contact EMV transactions, does `read_method` get set in the post response?

Reply:
The `read_method` param is only used for contactless payments.